src/App.jsx

```diff
@@ -1,107 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+      || null;
+}
+
+function getCommentById(postId) {
```

Comment:
```suggestion
function getCommentsById(postId) {
```
if you using filter it may return you the array with elements, not just one comment